website/docs/installing-podman-desktop/alternate-installation-methods.md

```diff
@@ -0,0 +1,235 @@
+---
+sidebar_position: 2
+title: Alternate installation methods.
+description: Consider installing Podman Desktop using one of the alternate installation method when you cannot use the default installation method.
+keywords: [podman desktop, containers, podman, installing, installation]
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Installing Podman Desktop with alternate installation methods
+
+Consider using one of these alternate installation method when you cannot use the [default installation method](/docs/installing-podman-desktop).
+
+<Tabs groupId="operating-systems">
+<TabItem value="win" label="Windows silent">
+
+#### Prerequisites
+
+- Windows 10 or 11.
+
+#### Procedure
+
+1. [Download the Windows installer](/downloads/windows).
+1. Run the PowerShell command:
+
+   ```shell-session
+   > Start-Process -ArgumentList "\S" -Wait -FilePath '.\podman-desktop-<version>-setup.exe'
```

Comment:
Other installation methods for Windows use the Windows shell and not Powershell. We should unify

Reply:
Nice idea indeed. In this pull request or a subsequent one?